ddtrace/contrib/openai/patch.py

```diff
@@ -244,6 +247,72 @@ def record_usage(self, span, usage):
             span.set_metric("openai.response.usage.%s_tokens" % token_type, num_tokens)
             self._statsd.distribution("tokens.%s" % token_type, num_tokens, tags=tags)
 
+    def generate_completion_llm_records(self, resp, span, args, kwargs):
+        # type: (Any, Span, List[Any], Dict[str, Any]) -> None
+        """Generate payloads for the LLM Obs API from a completion."""
+        if not self._config.llmobs_enabled:
+            return
+        choices = resp.choices
+        n = kwargs.get("n", 1)
+        prompt = kwargs.get("prompt", "")
```

Comment:
nit: We should initialize prompt to an array if we can

```suggestion
        prompt = kwargs.get("prompt", [""])
```